components/AppBar/examples/AppBarWrappingUITableViewControllerExample.m

```diff
@@ -0,0 +1,101 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+
+#import "MaterialAppBar.h"
+#import "MaterialAppBar+ColorThemer.h"
+#import "MaterialAppBar+TypographyThemer.h"
+
+@interface AppBarWrappingUITableViewControllerExample : UIViewController <UITableViewDataSource>
+
+@property(nonatomic, strong) MDCAppBarContainerViewController *appBarContainerViewController;
+@property(nonatomic, strong) MDCSemanticColorScheme *colorScheme;
+@property(nonatomic, strong) MDCTypographyScheme *typographyScheme;
+
+@end
+
+@implementation AppBarWrappingUITableViewControllerExample
+
+- (void)viewDidLoad {
+  [super viewDidLoad];
+
+  UITableViewController *tableViewController =
+      [[UITableViewController alloc] initWithStyle:UITableViewStylePlain];
+  tableViewController.title = @"Wrapped table view";
+  self.appBarContainerViewController =
+      [[MDCAppBarContainerViewController alloc] initWithContentViewController:tableViewController];
+
+  tableViewController.tableView.dataSource = self;
+  [tableViewController.tableView registerClass:[UITableViewCell class]
+                        forCellReuseIdentifier:@"cell"];
+
+  self.appBarContainerViewController.appBar.headerViewController.headerView.trackingScrollView =
+      tableViewController.tableView;
+
+  [MDCAppBarColorThemer applySemanticColorScheme:self.colorScheme
```

Comment:
please set
```
    self.colorScheme = [[MDCSemanticColorScheme alloc] init];
    self.typographyScheme = [[MDCTypographyScheme alloc] init];
```
in init, or otherwise they would be nil on non catalog targets (i.e. dragons)

Reply:
Done.